### ISSUE TITLE
Sending info for both grant types in Credential Offer

### ISSUE THREAD

**Issue by paulbastian** [openid/OpenID4VCI#114]
In the current specification, we have this example for Credential Offer ([link](https://openid.github.io/OpenID4VCI/openid-4-verifiable-credential-issuance-wg-draft.html#section-4.1.1-5)):

```
{
   "credential_issuer": "https://credential-issuer.example.com",
   "credentials": [
      "UniversityDegreeCredential",
      "org.iso.18013.5.1.mDL"
   ],
   "grants": {
      "authorization_code": {
         "issuer_state": "eyJhbGciOiJSU0Et...FYUaBy"
      },
      "urn:ietf:params:oauth:grant-type:pre-authorized_code": {
         "pre-authorized_code": "adhjhdjajkdkhjhdj",
         "user_pin_required": true
      }
   }
}
```

I could not think of any possible scenario, where it makes sense to have both grant types in one credential offer, either I'm already authenticated or I'm not? Therefore I propose to delete one of the grants from the example.

**Comment by tlodderstedt** [comment]
The primary reason for allowing for multiple grants is interoperability. If the wallet is free to implement either authz code or pre-authz code, the issuer must support both and leave the decision to the wallet.

**Comment by paulbastian** [comment]
I'm not against supporting multiple grant types in credential offer in general. In just saying this particular example/combination seems weird and confusing. Can anyone sketch a use case for this?

**Comment by paulbastian** [comment]
@Sakurann so you agree that examples should only use either auth code or preauth code flow?

**Comment by paulbastian** [comment]
This was resolved in #121
